### ISSUE TITLE
Login Page throws error

### ISSUE THREAD

**Issue by adrian-szwejk** [SiGMobileUIUC/car_pool_app#1]
When testing app out and trying to login without filling anything out, it would show a popup saying "[firebase_auth/unknown] Given String is smpty or Null instead of not letting the user try to login until they've filled out both TextFields. Same issues for emails that aren't formatted properly: "test" intead of "test@gmail.com" and for when format is valid but user doesn't exist.

**Comment by ashah305** [comment]
Hey, I made it so you have to make an account with the proper email base on the sign up page and then from there in order to login you have to type the email and password -- is that what you mean is the issue? I think an error will show up in any other case.

**Comment by adrian-szwejk** [comment]
> 

Yeah, I fixed the formatting of the email text field with some regexp functions to test string formatting however you can still make a request with the incorrect format (At least when I pushed), so I think we gotta a boolean variable or someway for the app to not make a request until the formatting is correct and even after we should make the UI nicer for why it failed (Like "Account doesn't exists, try again")
